app/views/session_timeout/_warning.html.erb

```diff
@@ -9,6 +9,9 @@
         <p id='<%= description_id %>'>
```

Comment:
Should we make the visual paragraph hidden to screen readers, so that there's not repetitive content?

```suggestion
        <p id="<%= description_id %>" aria-hidden="true">
```

Reply:
The screen reader message is a bit shorter, the thinking was we get the full message once and a shorter message every 20 seconds after. That's why I'd rather keep it visible. Unless there is a better way to accomplish that.

Reply:
Oh! I didn't see that they are different texts. I agree, I think they're different enough that having both conveyed to a screen-reader user isn't unnecessarily confusing, so should be fine to keep as-is.